### PR TITLE
[WIP] Add small ML like test case

### PR DESCRIPTION
**Pull request by jrapin**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context / Related issue

Add a small benchmark corresponding to small hyperparameter search

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the **CONTRIBUTING** document and completed the CLA (see **CONTRIBUTING**).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: nevergrad/optimization/differentialevolution.py [hunk 1]
```diff
@@ -81,8 +81,10 @@ def __init__(
         # config
         self._config = DifferentialEvolution() if config is None else config
         self.scale = float(1. / np.sqrt(self.dimension)) if isinstance(self._config.scale, str) else self._config.scale
-        pop_choice = {"standard": 0, "dimension": self.dimension + 1, "large": 7 * self.dimension}
+        pop_choice = {"standard": 0, "dimension": self.dimension + 1, "large": 7 * self.dimension, "sqrt": 100}
```
Comment (teytaud) [inline nevergrad/optimization/differentialevolution.py:84]: maybe you can tell us a bit more about this "sqrt" ?

**Comment by teytaud** [comment]
Is this PR still alive ?
Makes sense at least to me -- the "small_ml" problem looks like a reasonable first approximation of what a ML hyperparameter tuning problem can be (though for sure it's super hard to cover such a wide field in 12 lines :-) ).

**Comment by teytaud** [comment]
Should we merge that one ?   (after fixing the conflicts, not a big deal)

**Comment by jrapin** [comment]
@teytaud I did not merge because I was thinking about removing the sqrt option, what do you think about it?

**Comment by jrapin** [comment]
Replaced by #787

**Pull request closed without merging**